CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-# Version 1.8.00
+# Version 1.7.10
 
 ### Bug fixes
 - MASFoundation's OTP channel selection was returning an internal error to the original request. The Mobile SDK now sends only cancellation of OTP process, and/or results of the original request. [DE366491]
+- The Mobile SDK was causing an error during device registration, "The device has already been registered." which prevents users to re-register the device using different credentials when share keychain was incorrectly configured. The Mobile SDK improves the device registration process to minimize this type of error. [DE369778]
```

Comment:
During device registration, the  Mobile SDK was causing an error: "The device has already been registered".  This happened when the share keychain was incorrectly configured, preventing users from re-registering the device using different credentials. The Mobile SDK improved the device registration process to minimize this kind of error.